atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1700,28 +1703,33 @@ private void commitWrites(TransactionService transactionService) {
                 // We check the pre-commit conditions first since they may operate similarly to read write conflict
                 // handling - we should check lock validity last to ensure that sweep hasn't affected the checks.
                 timedAndTraced("userPreCommitCondition", () -> throwIfPreCommitConditionInvalid(commitTimestamp));
-                timedAndTraced("preCommitLockCheck", () -> throwIfImmutableTsOrCommitLocksExpired(commitLocksToken));
+                traced("preCommitLockCheck", () -> throwIfImmutableTsOrCommitLocksExpired(commitLocksToken));
```

Comment:
You've mentioned:
"refreshing commit locks: consider ConjureTimelockServiceBlocking.refreshLockLeases"
in the changelog, but I don't see a comment here in the code (where you have for others that are no longer timed). Is it perhaps adding a comment?

Reply:
Yep, I'll add one. I think I left this out as I thought it was obvious, but probably worth doing for the sake of completeness